addons/l10n_ca/models/res_partner.py

```diff
@@ -1,8 +1,7 @@
-# coding: utf-8
-from odoo import api, fields, models, _
+from odoo import fields, models
 
 
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
-    l10n_ca_pst = fields.Char('PST')
+    l10n_ca_pst = fields.Char(string='PST number', help='Canadian Provincial Tax Identification Number')
```

Comment:
I think the "normal" way to handle that is to define a `vat_label` on the company

Reply:
Yes but PST isn't VAT, and it's a different number than the VAT one